src/Workspaces/VisualBasic/Portable/Recommendations/VisualBasicRecommendationService.vb

```diff
@@ -71,6 +73,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Recommendations
             Return SpecializedCollections.EmptyEnumerable(Of ISymbol)()
         End Function
 
+        Private Function GetSymbolsForGlobalStatementContext(
+            context As VisualBasicSyntaxContext,
+            cancellationToken As CancellationToken
+        ) As IEnumerable(Of ISymbol)
```

Comment:
In the compiler, we would use `ImmutableArray(Of ISymbol)`.  I'm not sure what the IDE convention is.